debian/copyright

```diff
@@ -0,0 +1,52 @@
+
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: paflib
+Source: https://github.com/leitao/paflib_deb
+
+Files: *
+Author:  Carlos Eduardo Seo (ceseo) <contact@carlosseo.com>
+Copyright: Copyright IBM Corp. 2013
```

Comment:
Only IBM should be listed as copyright owner, so I suggest you change this line to:

    Copyright: Copyright IBM Corp. 2013-2015